lib/ansible/modules/remote_management/manageiq/manageiq_group.py

```diff
@@ -0,0 +1,596 @@
+#!/usr/bin/python
+#
+# (c) 2018, Evert Mulder <evertmulder@gmail.com> (base on manageiq_user.py by Daniel Korn <korndaniel1@gmail.com>)
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+
+module: manageiq_group
+
+short_description: Management of groups in ManageIQ.
+extends_documentation_fragment: manageiq
+version_added: '2.8'
+author: Evert Mulder
+description:
+  - The manageiq_group module supports adding, updating and deleting groups in ManageIQ.
+
+options:
+  state:
+    description:
+    - absent - group should not exist, present - group should be.
+    choices: ['absent', 'present']
+    default: 'present'
+  description:
+    description:
+    - The group description.
+    required: true
+    default: null
+  role_id:
+    description:
+    - The the group role id
+    required: false
+    default: null
+  role:
+    description:
+    - The the group role name
+    - The C(role_id) has president over the C(role) when supplied.
+    required: false
+    default: null
+  tenant_id:
+    description:
+    - The tenant for the group identified by the tenant id.
+    required: false
+    default: null
+  tenant:
+    description:
+    - The tenant for the group identified by the tenant name.
+    - The C(tenant_id) has president over the C(tenant) when supplied.
+    - Tenant names are case sensitive.
+    required: false
+    default: null
+  managed_filters:
+    description: A list of list of tag categories
```

Comment:
Can you add some more detail on what the structure means?  How is [[a, b, c, d]] different from [[a, b], [c, d]]?

Reply:
Actually with the example you updated, this text is good enough. However we explain it, it won't be clear until one looks at examples...  Perhaps slight tweak "A list of list of tags, grouped by category".

Reply:
I will use your suggestion and update the description.